tests/test_default_gaussian.py

```diff
@@ -625,7 +625,7 @@ def test_unsupported_gates(self, g, gaussian_device_3_wires):
         """Test error is raised with unsupported gates"""
         # TODO: Refactor this test to remove dynamic attribute loading, and instead
         # have separate tests for gates with specific number of parameters/wires.
-        if g == "BasisState":
+        if g == "BasisState" or g == "QubitStateVector":
             pytest.skip("Test not set up properly for gates with array parameters, needs refactoring.")
```

Comment:
We definitely need to refactor this test soon 😟

Reply:
Totally! Was weird to introduce a new decomposition for a qubit op and then have to refactor `test_default_gaussian.py`...